website/config.yaml

```diff
@@ -18,7 +18,8 @@ PygmentsCodeFences: true
 
 # Site-level parameters available through .Site.Params.{key}
 params:
-  author: Luc Perkins
+  author: Twitter, Inc.
+  description: A realtime, distributed, fault-tolerant stream processing engine from Twitter
   versions:
     heron: 0.13.7
```

Comment:
@lucperkins - why the heron version is hard coded. It might be hard to maintain this.